iceoryx_posh/test/moduletests/test_roudi_shm.cpp

```diff
@@ -287,28 +300,31 @@ TEST_F(PortManager_test, InterfaceAndApplicationsOverflow)
             });
 
         errorHandlerCalled = false;
-        auto interp = m_shmManager->acquireInterfacePortData(iox::capro::Interfaces::INTERNAL, "/itfPenguin");
-        EXPECT_THAT(interp, Eq(nullptr));
+        auto interfacePointer = m_shmManager->acquireInterfacePortData(iox::capro::Interfaces::INTERNAL, "/itfPenguin");
+        EXPECT_THAT(interfacePointer, Eq(nullptr));
         EXPECT_TRUE(errorHandlerCalled);
 
         errorHandlerCalled = false;
-        auto appp = m_shmManager->acquireApplicationPortData("/appPenguin");
-        EXPECT_THAT(appp, Eq(nullptr));
+        auto appPointer = m_shmManager->acquireApplicationPortData("/appPenguin");
+        EXPECT_THAT(appPointer, Eq(nullptr));
         EXPECT_TRUE(errorHandlerCalled);
     }
 
     // delete one and add one should be possible now
     {
         unsigned int testi = 0;
-        m_shmManager->deletePortsOfProcess(itf + std::to_string(testi));
-        m_shmManager->deletePortsOfProcess(app + std::to_string(testi));
-
-        auto interp =
-            m_shmManager->acquireInterfacePortData(iox::capro::Interfaces::INTERNAL, itf + std::to_string(testi));
-        EXPECT_THAT(interp, Ne(nullptr));
-
-        auto appp = m_shmManager->acquireApplicationPortData(app + std::to_string(testi));
-        EXPECT_THAT(appp, Ne(nullptr));
+        auto newItfName = itf + std::to_string(testi);
```

Comment:
Could we not directly create the CString100 here instead of constructing a Cstring100 from these later? newItfName is not used anywhere else to be required of type std::string.

Reply:
Nope, currently `cxx::string::operator+()` is missing :-(